package.json

```diff
@@ -22,7 +22,8 @@
     "es6-module-loader": "0.14.0",
     "mocha": "~2.0.0",
     "chai": "^1.10.0",
-    "react-tools": "^0.12.1"
+    "react-tools": "^0.12.1",
+    "atob": "^1.1.2"
```

Comment:
Are you sure we want to get on the wrong side of the Modularity Wars?! 

Thanks, will drop `atob`.